jsx_test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Card from "./card";
+
+export default class CardList extends React.Component {
```

Comment:
Component should use createClass instead of es6 class                          react/prefer-es6-class